### PR TITLE
Update  4.2 specs changelog

### LINKED ISSUE #693: Update 4.2, 4.3 and 5.0 SPECs to add new 4.x releases
Hello,

I have detected that 4.2 and further SPECs don't have all changes inside changelog. We have to update this.

### PR DESCRIPTION
**Pull request by jcruzlp**
|Related issue|
|---|
|Closes #693 |

## Description

This PR does the same as https://github.com/wazuh/wazuh-packages/pull/760 but adding the required changes to 4.2

### CODE REVIEW

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes, gj.

File: debs/SPECS/4.2.0/wazuh-agent/debian/changelog [hunk 1]
```diff
@@ -4,6 +4,30 @@ wazuh-agent (4.2.0-RELEASE) stable; urgency=low
 
  -- Wazuh, Inc <info@wazuh.com>  Mon, 26 Apr 2021 11:51:55 +0000
 
+wazuh-agent (4.1.4-RELEASE) stable; urgency=low
```
[the remark this answers is not in the document]
Comment (jcruzlp) [reply debs/SPECS/4.2.0/wazuh-agent/debian/changelog:7]: Fixed here: https://github.com/wazuh/wazuh-packages/pull/761/commits/c3b17f96c1628628f9c4555e1e8c6c6c47d51163

File: debs/SPECS/4.2.0/wazuh-manager/debian/changelog [hunk 2]
```diff
@@ -4,6 +4,31 @@ wazuh-manager (4.2.0-RELEASE) stable; urgency=low
 
  -- Wazuh, Inc <info@wazuh.com>  Mon, 26 Apr 2021 11:51:55 +0000
 
+
+wazuh-agent (4.1.4-RELEASE) stable; urgency=low
```
[the remark this answers is not in the document]
Comment (jcruzlp) [reply debs/SPECS/4.2.0/wazuh-manager/debian/changelog:13]: Fixed here: https://github.com/wazuh/wazuh-packages/pull/761/commits/c3b17f96c1628628f9c4555e1e8c6c6c47d51163

**Review by alberpilot — APPROVED** [review]
LGTM

File: debs/SPECS/4.2.0/wazuh-manager/debian/changelog [hunk 3]
```diff
@@ -4,6 +4,36 @@ wazuh-manager (4.2.0-RELEASE) stable; urgency=low
...
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
+
+wazuh-agent (4.1.1-RELEASE) stable; urgency=low
+
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Fri, 05 Mar 2021 13:24:41 +0000
+
```
Comment (okynos) [inline debs/SPECS/4.2.0/wazuh-manager/debian/changelog:36]: Add 3.13.3 to the versioning, thanks.

**Review by okynos — CHANGES_REQUESTED** [review]
Minor change, ty.

File: debs/SPECS/4.2.0/wazuh-manager/debian/changelog (hunk 3 above)
Comment (jcruzlp) [reply debs/SPECS/4.2.0/wazuh-manager/debian/changelog:36]: Fixed here: https://github.com/wazuh/wazuh-packages/pull/761/commits/e4f1ac9588d3fa88e9c9241466a33ba4cc0284c5 and https://github.com/wazuh/wazuh-packages/pull/761/commits/8dbb9e320792dac67bb944d8e27e9a3706724fa0

**Review by okynos — APPROVED** [review]
LGTM.

**Pull request merged**